### ISSUE TITLE
Markdown table formatting

### ISSUE THREAD

**Issue by spapastamkou** [programminghistorian/jekyll#2503]
In a [new FR lesson under review](https://github.com/programminghistorian/ph-submissions/issues/459), I have a problem of not viewing correctly a wide table in the previsualisation (and future publication I suppose) : one part of it is lost when viewing the page (second table [here](http://programminghistorian.github.io/ph-submissions/fr/lecons/concevoir-recherche-donnees-nodegoat), paragraphs 15-16). 
I want to insert a scroll bar so that the table is scrollable horizontally but so far I managed to insert a bar and have a raw table instead, like this:

![nodegoat-table](https://user-images.githubusercontent.com/27706167/156220836-b3536971-d1fa-415c-86e5-676573853c33.png)

For the above result I was inspired by [this solution](https://mkang32.github.io/2020/11/15/Adding-scroll-to-overflowing-table.html) but without touching the PH CSS. 

My last hope is to add a html block and reformat the table with html tags; but before doing this, if someone has already managed to do what I try to do here keeping the markdown format, please let me know. Thanks!

**Comment by spapastamkou** [comment]
@anisa-hawes @jenniferisasi @programminghistorian/french-team @rivaquiroga maybe??

**Comment by jenniferisasi** [comment]
let me check on the lesson file... checked. 

Ok, we don't have a table-wrapper in our [CSS](https://raw.githubusercontent.com/programminghistorian/jekyll/gh-pages/css/style.css) to begin with, so I believe we have to first add one and then you should be able to use the HTML route with a div that calls to the table-wrapper. 

@programminghistorian/technical-team on it.

**Comment by spapastamkou** [comment]
I tried once more after the CSS change on `ph-submissions` by adding the table between 
```<html>
<div class="table-wrapper" markdown="block">

[table]

</div>
</html>
```

but still not working... Thank you so very much @jenniferisasi, sincerely! Tomorrow is another day, I'll try again:)

**Comment by acrymble** [comment]
Since our tutorials are really just a proof of principle, is it possible to reduce the size of the data being used in the tutorial to avoid this issue?

**Comment by spapastamkou** [comment]
@acrymble ... or even to just describe in the text this table's data as the table illustrates what we should avoid end up with. I'll discuss these options with the author who is currently revising.

**Comment by ZoeLeBlanc** [comment]
Just fyi, now have a PR in the ph-submissions repo working on this issue https://github.com/programminghistorian/ph-submissions/pull/466

**Comment by anisa-hawes** [comment]
This Issue can be closed, see #2504.

**Comment by spapastamkou** [comment]
The table is now scrollable, thanks a lot all of for the help.
